 .nvmrc

```diff
@@ -0,0 +1 @@
+22.14.0
```

Comment:
please remove the `npmrc` file. This change will pin the node version - which can be a good thing - but to have consistency across all spacecat repos, let's not introduce it here.